jena-arq/src/main/java/org/apache/jena/riot/system/Checker.java

```diff
@@ -40,19 +40,30 @@
 /**
  * Functions for checking nodes, triples and quads.
  * <p>
- * If the errorHandler is null, use the system wide handler.
+ * The check functions have two basic signatures:<br>
+ * 1. "<tt>check...(<i>object</i>)</tt>"
+ * 2. "<tt>check...(<i>object, errorHandler, line, col</i>)</tt>"
+ * The first type are for boolean testing and do not generate output. They call the
+ * second type with default values for the last 3 parameters: nullErrorHandler, -1L, -1
+ * The second type are for boolean testing and optionally generate error handling
+ * output.
  * <p>
- * If the errorHandler line/columns numbers are -1, -1, messages do not include them.
+ * Parameters:<br>
+ * {@link ErrorHandler} errorHandler - the errorHandler for output. If the errorHandler
+ * is null, use the system wide handler.<br>
+ * {@link long} line - code line number generating the check.<br>
```

Comment:
`long` isn't a class so the link does not work (I get a warning in Eclipse). `Long` would be incorrect.